packages/components/src/Heading/Heading.stories.tsx

```diff
@@ -55,3 +55,10 @@ Heading4ColorNeutral.args = {
   color: "neutral",
   size: "h4",
 };
+
+export const Heading4ColorCurrent = Template.bind(null);
+Heading4ColorCurrent.args = {
+  children: "Current color Heading 4",
+  color: "currentColor",
```

Comment:
this isnt really testing anything/not sure that we really need a story, but this was just so i could see a) that currentColor was being set correctly and b) you can see the color difference when you switch to color being `base` :(